src/use_case/add_project/AddProjectInteractor

```diff
@@ -0,0 +1,122 @@
+package use_case.add_project;
+import entity.Project;
+import entity.ProjectFactory;
+
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class AddProjectInteractor implements AddProjectInputBoundary {
+    final AddProjectDataAccessInterface userDataAccessObject;
```

Comment:
SUGGESTION: change the visibility of these interactor attributes to private. Also, review other files to confirm visibility of each attribute.

Reply:
Thank you! Great catch. These have been implemented and pushed.